pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -43,17 +85,61 @@ func init() {
 	aliasFormat = regexp.MustCompile(`\{\{\s*(.+?)\s*\}\}`)
 }
 
+func (e *CloudWatchExecutor) AlertQuery(logsClient *cloudwatchlogs.CloudWatchLogs, ctx context.Context, queryContext *tsdb.TsdbQuery) (*cloudwatchlogs.GetQueryResultsOutput, error) {
+	const MaxAttempts = 5
+	const PollPeriod = 500 * time.Millisecond
```

Comment:
I think these variables shouldn't be capitalized, since capitalization in Go means to export symbols. That doesn't apply here since the variables are function local, but I don't see any reason to diverge from normal naming:
```suggestion
	const maxAttempts = 5
	const pollPeriod = 500 * time.Millisecond
```